### DISCUSSION TITLE
ERROR:Create Strapi project

### DISCUSSION THREAD

**Discussion by rushikesh-19** [strapi/strapi#7728] [Legacy Topics]
A:\blog-strapi\blog-strapi>yarn create strapi-app backend
yarn create v1.22.5
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...
success Installed "create-strapi-app@3.1.4" with binaries:
      - create-strapi-app
'C:\Users\Rushikesh' is not recognized as an internal or external command,
operable program or batch file.
error Command failed.
Exit code: 1
Command: C:\Users\Rushikesh Gadekar\AppData\Local\Yarn\bin\create-strapi-app
Arguments: backend
Directory: A:\blog-strapi\blog-strapi
Output:

**Comment by derrickmehaffy** [comment]
Can you please provide which version of node you are using?

**Reply by rushikesh-19** [reply]
12.18.3

**Reply by derrickmehaffy** [reply]
I was not able to reproduce this on my own Windows 10 system. :thinking: can you try with npx `npx create-strapi-app@latest test-project --quickstart`
